src/main/java/com/bookclub/model/Event.java

```diff
@@ -6,18 +6,30 @@ public class Event {
     private String organizer;
     private String date;
     private String location;
+    private Integer eventid;
+    private String description;
 
-    public Event(String title, String organizer, String date, String location) {
+    public Event(Integer eventid, String title, String organizer, String date, String location, String description) {
```

Comment:
Same here, treat ID and description as optional for the time being, so not required when constructing through the constructor we have 😅 

```suggestion
    public Event(String title, String organizer, String date, String location) {
```